suite-native/app/src/navigation/AppTabNavigator.tsx

```diff
@@ -14,20 +15,22 @@ import { rootTabsOptions } from './routes';
 const Tab = createBottomTabNavigator<AppTabsParamList>();
 
 export const AppTabNavigator = () => (
-    <Tab.Navigator
-        initialRouteName={AppTabsRoutes.HomeStack}
-        screenOptions={{
-            headerShown: false,
-            unmountOnBlur: false,
-        }}
-        tabBar={(props: BottomTabBarProps) => (
-            <TabBar tabItemOptions={rootTabsOptions} {...props} />
-        )}
-    >
-        <Tab.Screen name={AppTabsRoutes.HomeStack} component={HomeStackNavigator} />
-        <Tab.Screen name={AppTabsRoutes.AccountsStack} component={AccountsStackNavigator} />
-        <Tab.Screen name={AppTabsRoutes.Action} component={ActionScreen} />
-        <Tab.Screen name={AppTabsRoutes.Prices} component={PricesScreen} />
-        <Tab.Screen name={AppTabsRoutes.SettingsStack} component={SettingsStackNavigator} />
-    </Tab.Navigator>
+    <>
+        <Tab.Navigator
+            initialRouteName={AppTabsRoutes.HomeStack}
+            screenOptions={{
+                headerShown: false,
+                unmountOnBlur: false,
+            }}
+            tabBar={(props: BottomTabBarProps) => (
+                <TabBar tabItemOptions={rootTabsOptions} {...props} />
```

Comment:
Could you just pass whole `SendReceiveBottomSheet` as prop here? Something like:

```tsx
<TabBar SendReceiveComponent={SendReceiveBottomSheet} ..... />
// later in TabBar
<SendReceiveComponent isVisible={...} onVisibilityChange={...} />
```

That should solve issue with circular dependency and you can even move Tabs back to navigation package. It will also remove need to import from module package `module-receive-send` in `tab-bar`.

Reply:
But this will cause the package `module-send-receive` will be a dependency of `navigation` package - and what if we wanted to navigate somewhere from module-send-receive later when this receive flow is prepared completely with all functionality (I mean post MVP stage)?

Reply:
It won't, module-receive-send is used as dependency only in app, component is passed as prop of TabBar from app package.

Reply:
Oh, I see. I was confused about the "navigation" word and at first, I read it, did not catch _app_ paclage and thought it is a package :)

suite-native/app/src/**navigation**/XY.tsx

vs. 

suite-native/**navigation**/XY.tsx

Reply:
95f10db594c20bc5c4a003de5a8d32397df004ed